OnlineLibrary.BLL/Services/AuthorService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using OnlineLibrary.Common.Entities;
+using OnlineLibrary.Common.Helpers;
+using OnlineLibrary.BLL.Interfaces;
+using OnlineLibrary.DAL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OnlineLibrary.Common.Validators;
+
+namespace OnlineLibrary.BLL.Services
+{
+    public class AuthorService : IAuthorService
+    {
+        private readonly IUnitOfWork unitOfWork;
+
+        private readonly AuthorValidator authorValidator;
+
+        public AuthorService(IUnitOfWork uow)
+        {
+            unitOfWork = uow;
+            authorValidator = new AuthorValidator();
+        }
+
+        public int CreateAuthor(Author author)
+        {
+            var results = authorValidator.Validate(author);
```

Comment:
You don't need to validate manually as you have already registered fluent validators, so before the author passed to the method it will be implicitly validated and in case of error will throw an exception 

So remove and verify that exception is thrown